package-lock.json

```diff
@@ -879,9 +879,8 @@
     },
     "node_modules/@babel/helper-plugin-utils": {
       "version": "7.16.7",
-      "resolved": "https://registry.npmjs.org/@babel/helper-plugin-utils/-/helper-plugin-utils-7.16.7.tgz",
```

Comment:
No dependencies were added/removed. Please revert all changes to this file.

Reply:
This was also introduces by @shreedhanu08 as far as I can see

Reply:
Right, but they don't exist in the main branch. So as the PR owner, _you_ are the driving contributor for _all_ of the changes in this PR - not just the ones that you made.

You'll need to either remove any changes that you didn't make, or take ownership of the changes that you're looking to get merged.

Reply:
@AV-MadsChris Please remove this from your PR.

Reply:
As a note I cannot be held responsible for code you have asked me to use. Any changes shree has made I will not take responsibility for